djangocms_snippet/migrations/0010_cms4_grouper_version_data_migration.py

```diff
@@ -0,0 +1,48 @@
+from django.db import migrations
+
+from djangocms_snippet.conf import DJANGOCMS_SNIPPET_VERSIONING_MIGRATION_USER_ID
+
+try:
+    from djangocms_versioning.constants import DRAFT
+    djangocms_versioning_enabled = True
+except:
```

Comment:
Plain exceptions are bad as they can catch and swallow other errors making debugging difficult. 

```suggestion
except ImportError:
```